docs/modules/develop/pages/permissions.adoc

```diff
@@ -0,0 +1,92 @@
+= Permissions
+
+Since Decidim has multiple roles, we needed a permissions system to discover what actions can a user perform, given their roles. The basis of the current permissions system were added on https://github.com/decidim/decidim/pull/3029[#3029], so be sure to check that PR (and the related ones) to read the discussion and the motivations behind the change.
+
+== Overview
+
+When checking for permission to perform an action, we check this chain:
+
+. The component permissions
+. The participatory space permissions
+. The core permissions
+
+This way we're going from more specific to more general.
+
+== Explanation
+
+We wrap the permission and its context in a `PermissionsAction` object. It also holds the state of the permission (whether it's been allowed or not).
+
+Each component and space must define a `Permissions` class, inheriting from `Decidim::DefaultPermissions`. The `Permissions` class must define a `permissions` instance method. this class will receive the permission action, and the `permissions` method must return the permission action. The `Permissions` class can set the action as allowed or disallowed.
+
+There's a small limitation in the permission action state machine: once it's been disallowed it can't be reallowed. This is to avoid mischievous modules modifying permissions.
+
+Permission actions have a scope. It's usually either `:public` or `:admin`, and the `Permissions` class usually handles the `:public` scope, while it delegates the `:admin` one to another specialized class.
+
```

Comment:
:tada: I think it was necessary to expand this section with some examples. Some time ago, I had to deal with permissions for an issue and this explanation would have been very helpful :+1:

Reply:
This file was actually already there!! It was in another file: `docs/advanced/add_authorizable_action.md`, so this confirms my hypothesis that it was difficult to find things in too many files.